src/Altinn.App.Core/Helpers/ObjectUtils.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+
+namespace Altinn.App.Core.Helpers;
+
+/// <summary>
+/// Utilities for working with model instances
+/// </summary>
+public static class ObjectUtils
+{
+    /// <summary>
+    /// Recursively initialize all <see cref="List{T}"/> properties on the object that are currently null
+    /// </summary>
+    /// <param name="model"></param>
+    public static void InitializeListsRecursively(object model)
```

Comment:
Think I need a walkthrough of the why. I believe you, but would be great to have an actual live demo and see the actual why :)
Also seems to me like this method does more than initialize List now